docs/deployment-guide.md

```diff
@@ -450,6 +450,39 @@ Once ACS has initialized access the system using the following URLs with a brows
 * Repository: `http://<nginx-host-ip>/alfresco`
 * API Explorer: `http://<nginx-host-ip>/api-explorer`
 
+### ACS cluster
+
+Due to load or high availability needs, you might want to deploy a cluster of several repository nodes. This can be achieved rather simply by:
+
+* Giving the playbook the location of the shared storage used for the ACS contentstore (See [Shared storage documentation](shared-contentstore.md) for details).
+* Specifying several hosts within the repository hosts group
+
+> :warning: as mention in the [Alfresco official documentation](https://docs.alfresco.com/content-services/latest/admin/cluster/#scenarioredundancycluster), "All the servers in a cluster should have static IP addresses assigned to them".
+> Not meetign this pre-requisite won't prevent the playbook from working but the cluster might will most likely stop working in case one of the server in the architecture changes IP address.
```

Comment:
```suggestion
> Not meeting this pre-requisite won't prevent the playbook from working but the cluster might will most likely stop working in case one of the server in the architecture changes IP address.
```